src/rewriters/Rewriter.h

```diff
@@ -14,7 +14,7 @@ class Rewriter {
 public:
     Rewriter(Logic & logic, TConfig & cfg) : logic(logic), cfg(cfg) {}
 
-    PTRef rewrite(PTRef root) {
+    virtual PTRef rewrite(PTRef root) {
```

Comment:
Why should this be virtual?

Reply:
I believe that it's because `DivModRewriter` also has a method called `PTRef rewrite(PTRef root)`.  That method should be marked `override`.  I'll add it to the PR and let's see then if it's correct.

Reply:
I see. OK, let's go with this.